controllers/requestController.js

```diff
@@ -95,23 +95,30 @@ module.exports = {
     },
 
     add: function (data) {
+        var addColor = 'red';
```

Comment:
This file contains many more colors being sent as strings, why not move them all into constants?